package.json

```diff
@@ -36,7 +36,7 @@
     "fuse.js": "^2.2.0",
     "glob": "^5.0.14",
     "highlight.js": "^8.9.1",
-    "linkifyjs": "2.0.0-beta.4",
+    "linkifyjs": "2.1.0",
```

Comment:
the reason this was pinned to that particular version was due to https://github.com/vector-im/vector-web/issues/2010 - can you check it doesn't reintroduce that regression?